demo/src/screens/realExamples/ProductPage/index.tsx

```diff
@@ -0,0 +1,138 @@
+  'https://images.pexels.com/photos/2872767/pexels-photo-2872767.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
+  'https://images.pexels.com/photos/3271010/pexels-photo-3271010.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
+  'https://images.pexels.com/photos/3206153/pexels-photo-3206153.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500'
+];
+
+class Product extends Component {
+  state = {
+    isColor: false,
+    isSize: false,
+    colorOption: colorOptions.red,
+    sizeOption: sizeOptions.s,
+  };
+  
+  pickOption(opt: any) {
+    this.setState({
+      colorOption: opt
+    });
+  }
+  
+  render() {
+    const {isColor, colorOption, isSize, sizeOption} = this.state;
+    
+    return (
+      <ScrollView>
+        <View>
+          <Image
+            style={{position: 'absolute', top: 10, right: 10, zIndex: 100, tintColor: Colors.white}}
+            source={require('../../../assets/icons/share.png')}/>
+          <Carousel
+            containerStyle={{height: 200}}
+            pageControlProps={{size: 6}}
+            pageControlPosition={Carousel.pageControlPositions.OVER}
+          >
+            {images.map((image, i) => {
+              return (
+                <View flex centerV key={i}>
+                  <Image
+                    overlayType={Image.overlayTypes.BOTTOM}
+                    style={{flex: 1}}
+                    source={{uri: image}}
+                  />
+                </View>
+              );
+            })}
+          </Carousel>
+        </View>
+    
+        <View paddingH-page>
+          <View row centerV>
+            <Text text40M marginT-s7>New Product</Text>
+          </View>
+          <Text text80 dark40 marginV-s1>SKU: 1234567890</Text>
+          <Text text50L marginV-s2>$55.00</Text>
+      
+          <View marginV-s2 paddingV-s2 style={{borderColor: Colors.dark70, borderBottomWidth: 1}}>
+            <Text dark30>Color</Text>
+            <TouchableOpacity row centerV spread onPress={()=> this.setState({isColor: true})}>
+              <View row centerV>
+                <View style={{width: 14, height: 14, borderRadius: 12, marginRight: 6}} backgroundColor={colorOption.color}/>
+                <Text dark10 text70>{colorOption.name}</Text>
+              </View>
+              <Image source={require('../../../assets/icons/chevronDown.png')}/>
+            </TouchableOpacity>
+          </View>
+          <ActionSheet
+            title={'Select a Color'}
+            options={[
+              {label: colorOptions.red.name, onPress: () => this.pickOption(colorOptions.red)},
+              {label: colorOptions.green.name, onPress: () => this.pickOption(colorOptions.green)},
+              {label: colorOptions.blue.name, onPress: () => this.pickOption(colorOptions.blue)},
+            ]}
+            visible={isColor}
+            onDismiss={()=> this.setState({isColor: false})}
+          />
```

Comment:
Why not use the Picker component?

Reply:
I thought that Picker doesn't have ActionSheet, but seems like it has in custom options

Reply:
Yes, technically you achieve the same behavior with custom item render.